packages/admin-next/dashboard/src/lib/client/products.ts

```diff
@@ -1,24 +1,20 @@
-import {
-  ProductDeleteRes,
-  ProductListRes,
-  ProductRes,
-} from "../../types/api-responses"
+import { ProductDeleteRes, ProductListRes } from "../../types/api-responses"
```

Comment:
suggestion: add http response types to `@medusajs/types`

Reply:
@olivermrbl I've done a lot of `any` for now, I'll add all types in one go once I get everything working